vdb-bench-assembly/plugins/vdb-bench-dataservice/dataservicePageController.js

```diff
@@ -133,7 +133,7 @@
             return ConnectionSelectionService.getConnectionState(conn);
         };
 
-        vm.selectPage(DSPageService.DS_HOME_PAGE);
+        vm.selectPage(DSPageService.DATASERVICE_SUMMARY_PAGE);
     }
```

Comment:
So far there's no emphasis or request to really have a home page or dashboard in DSB.. focus has been on designing a start-up experience that's as simple as possible.

So by moving the **DS Home** to the bottom tab and calling it a **Dashboard** keeps in the UI, but de-emphasizes it for the time being.  So making the DS summary page the starting point addresses the **simple** entry point.

I fully expect to there'll be multiple iterations as we move forward.